addons/jrm/XEH_preInit.sqf

```diff
@@ -4,4 +4,8 @@ ADDON = false;
 
 #include "XEH_PREP.hpp"
 
+[QGVAR(forceRespawn), {
+    call FUNC(forceRespawn);
+}] call CBA_fnc_addEventHandler;
```

Comment:
_this is kept in scope or? what about remove the whole code block

Reply:
`_this` is passed into the function implicit. basically I just wanted an eventhandler to force respawn people without the need of `remoteExec`.

Reply:
then it could just be 
```
[QGVAR(forceRespawn), FUNC(forceRespawn)] call CBA_fnc_addEventHandler;
```

but yeah it's fine

Reply:
Yes, i prefer having it wrapped, so there is less code saved within the internal hashmap/array of the cba framework. makes it imo better to read once you have to check what eventhandlers are added to that id.